spockbot/mcdata/items.py

```diff
@@ -0,0 +1,67 @@
+import sys
+
+from minecraft_data.v1_8 import items_list
+
+from spockbot.mcdata.utils import camel_case, find_by
+
+items = {}
+items_name = {}
+
+
+def get_item(item, meta=None, init=True):
+    ret = None
+    if isinstance(item, int):  # by id
+        ret = find_by(item, items)
+    else:  # by name
+        ret = find_by(item, items_name)
+    if init and ret is not None:
+        return ret(meta)
+    return ret
+
+
+class Item(object):
+    id = -1,
+    display_name = "Item",
+    stack_size = 0,
+    name = "item",
+    variations = {}
+
+    def __init__(self, meta=None):
+        if meta is None:
+            return
+        self.metadata = meta
+        if self.metadata in self.variations:
+            self.display_name = self.variations[self.metadata]["display_name"]
```

Comment:
All of the variation's changes should be applied, not just the name. In the future, there might be other things that vary.